packages/api-request-builder/test/default-params.spec.js

```diff
@@ -3,7 +3,11 @@ import { setDefaultParams } from '../src/default-params'
 
 describe('defaultParams', () => {
   it('should set default params for a normal endpoint', () => {
-    const serviceFeatures = [features.query, features.queryOne]
+    const serviceFeatures = [
```

Comment:
Instead of modifying this test, it'll be better to write a new test case. Something like:
```js
  it('should set default params for shipping-methods', () => {
    const serviceFeatures = [
      features.query,
      features.queryOne,
      features.queryExpand,
      features.queryLocation,
    ]
    const params = {}
    setDefaultParams.call({ features: serviceFeatures, params })
    expect(params).toEqual({
      id: null,
      expand: [],
      location: {
        country: null,
        currency: null,
        state: null,
      },
      pagination: {
        page: null,
        perPage: null,
        sort: [],
      },
      query: {
        operator: 'and',
        where: [],
      },
    })
  })
```

Reply:
I can only recommend not to have these super big `toEqual` expectations. Rather split them up into

desc: params
it: should contain id
it: should contain expand
it: should contain location
...

And use `expect.objectContaining` to just specify what you expect.

Reply:
added here https://github.com/commercetools/nodejs/pull/381/commits/d86d95a6e56544a34f4251ebc8b019391699cc57#diff-a63965709a1d0dead073c9d0e25e9c37R99